documentation/trunk/solutions/basic-computing/eiffelbase/eiffelbase-tutorial/eiffelbase-kernel.wiki

```diff
@@ -102,7 +102,6 @@ These two features enable clients to ascertain the dynamic type of an entity at
 
 ==Miscellaneous==
 
-The query Void, of type <eiffel>NONE</eiffel>, denotes a reference that is always void - not attached to any object. <br/>
```

Comment:
I guess, this text related to Void should be kept. 
This explains what is "Void" and its type.